cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/SteadyStateHypothesisExport.java

```diff
@@ -184,6 +185,7 @@ private static String cgmesTapChangerId(TwoWindingsTransformer twt, String tapCh
 
     private static void writeTapChangers(Network network, String cimNamespace, Map<String, List<RegulatingControlView>> regulatingControlViews, XMLStreamWriter writer, CgmesExportContext context) throws XMLStreamException {
         for (TwoWindingsTransformer twt : network.getTwoWindingsTransformers()) {
+            CgmesExportUtil.addUpdateCgmesTapChangerExtension(twt, context);
             if (twt.hasPhaseTapChanger()) {
```

Comment:
I suppose that you added to be able to export the SSH file without exporting the EQ file.
You also have to add this method some lines below for the threeWindingsTransformers.

Reply:
Thanks, I'll update that!